bin/brew

```diff
@@ -161,7 +168,7 @@ then
   # Hide shellcheck complaint:
   # shellcheck source=/dev/null
   source "$HOMEBREW_BASH_COMMAND"
-  { "$HOMEBREW_COMMAND" "$@"; exit $?; }
+  { "brew-$HOMEBREW_COMMAND" "$@"; exit $?; }
```

Comment:
:cool:. Mind naming this `homebrew-$HOMEBREW_COMMAND` to make the namespacing a bit more similar?